docs/src/upgrade/release-3.7.x.asciidoc

```diff
@@ -341,7 +341,7 @@ See: link:https://tinkerpop.apache.org/docs/x.y.z/reference/#dateAdd-step[dateAd
 See: link:https://tinkerpop.apache.org/docs/x.y.z/reference/#dateDiff-step[dateDiff()-step]
 See: link:https://issues.apache.org/jira/browse/TINKERPOP-2979[TINKERPOP-2979]
 
-===== `datetime()` for current server time
+===== `datetime()` for Current Server Time
 
 Function `datetime()` extended to return current server time when used without argument.
```

Comment:
Do we want to add a note that datetime() remains a script only function? Or is it best to leave that out?